src/main/java/io/crate/testing/CrateTestCluster.java

```diff
@@ -148,6 +152,10 @@ public Builder numberOfNodes(int numberOfNodes) {
                 throw new IllegalArgumentException(String.format("invalid number of nodes: %d", numberOfNodes));
             }
             this.numberOfNodes = numberOfNodes;
+            if (settings.isEmpty()) {
+                settings = new HashMap<>();
+            }
+            settings.put("node.max_local_storage_nodes", numberOfNodes);
```

Comment:
is this safe for CrateDB < 3.0 ?

Reply:
Yes, it's safe, tested with versions 1.0.x up to 3.0.x